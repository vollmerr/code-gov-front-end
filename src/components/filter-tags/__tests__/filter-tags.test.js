import React from 'react';

import FilterTags from 'components/filter-tags'
import FilterTag from 'components/filter-tag'

import { shallow } from 'enzyme';

describe('Testing <FilterTags /> component', () => {
  const props ={
    onClick: jest.fn(),
    filters: [
      {
        category: 'test-category',
        modified: 1111111111111,
        value:'test-value',
        title: 'test-title'
      }
    ]
  }
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<FilterTags {...props}/>);
  });
  it('renders 1 Autocomplete with to', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('tests onClick', () => {
    wrapper.find(FilterTag).prop('onClick')();
    expect(props.onClick).toBeCalled();
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})