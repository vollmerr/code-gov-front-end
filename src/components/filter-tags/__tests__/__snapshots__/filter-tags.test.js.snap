// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FilterTags /> component should render correct 1`] = `
<div
  className="filter-tags"
>
  <FilterTag
    category="test-category"
    key="test-category:test-value"
    onClick={[MockFunction]}
    title="test-title"
    value="test-value"
  />
</div>
`;
